recipes/rust-bio-tools/build.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash -e
 
+if [ `uname` == Darwin ]; then
+    export MACOSX_DEPLOYMENT_TARGET=10.14
```

Comment:
@dpryan79 @daler @epruesse @bgruening do we want to target later OSXs? I.e. do we guarantee a minimum compatible OSX version somewhere which we would break this way?

Reply:
The CI infrastructure sets its own OSX target version. That was copied from conda-forge, but perhaps they're using a more recent target these days. I know at least previously people used to update OSX pretty religiously, but I don't know how accurate that is these days. If the 10.12 (or whatever we target by default) footprint is miniscule I would definitely support changing to something more recent.